Admin/PageAdmin.php

```diff
@@ -45,6 +51,7 @@ protected function configureListFields(ListMapper $listMapper)
     protected function configureFormFields(FormMapper $formMapper)
     {
         parent::configureFormFields($formMapper);
+        $ivoryCKeditorAvailable = class_exists('Ivory\CKEditorBundle\IvoryCKEditorBundle') && $this->ivoryCkeditor;
```

Comment:
I would also prefer a solution, where the existence check is done in the extension 

Options set + bundle there => fine 
Options set + bundle not there => exceprion

I think WE did some equal with the burgov-bundle in CmfSeo.